### DISCUSSION TITLE
Hey there :3

### DISCUSSION THREAD

**Discussion by JoshuyPan** [Cyfrin/foundry-full-course-cu#2088] [General]
Hey guys, it's Josh here and i'm in the blockchain development from almost 2 months. 

I'm looking for friend/s to code with so we can learn faster together 🥹
If you want to adopt a cute blockchain development trainee, feel free to comment below ❤️

**Comment by CadetTalha** [comment]
hey mate
i have completed blockchain basics four days ago and now started learning about solidity basics 
basically im undergraduate student of AI, we can work together to learn and code together

**Comment by EngrPips** [comment]
Awesome, I hope you find a buddy soon.

**Reply by JoshuyPan** [reply]
i don't think so, but its oki hehe
